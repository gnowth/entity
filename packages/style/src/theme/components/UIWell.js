import { css } from 'styled-components';

import { color } from '../../selectors';

export const component_uiWell_danger = css`
  color: ${color({ name: 'danger' })};
  background-color: ${color({ name: 'danger', weight: '50' })}
`;

export const component_uiWell_primary = css`
  color: ${color({ name: 'primary' })};
`;

export const component_uiWell_page = css`
  border: 0;
`;

export const component_uiWell_page_flat = css`
  border: 0;
  border-radius: 0;
`;
