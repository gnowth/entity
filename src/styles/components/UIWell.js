import { css } from 'styled-components';
import { lighten } from 'polished';

export const danger = css`
  color: ${props => props.theme.vars.colorDanger};
  background-color: ${props => lighten(0.275, props.theme.vars.colorDanger)}
`;

export const primary = css`
  color: ${props => props.theme.vars.colorPrimary};
`;

export const page = css`
  border: 0;
`;

export const page_flat = css`
  border: 0;
  border-radius: 0;
`;
