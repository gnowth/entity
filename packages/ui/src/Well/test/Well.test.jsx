import renderer from 'react-test-renderer';
import React from 'react';

import UIWell from '..';

it('renders correctly', () => {
  const tree = renderer.create(
    <UIWell />,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
