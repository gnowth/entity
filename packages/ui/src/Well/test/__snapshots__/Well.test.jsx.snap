// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Well__UIWell-xiuhp8-0 gqTYNb"
/>
`;
